phoenicis-javafx/src/main/java/com/playonlinux/javafx/views/mainwindow/console/ConsoleTab.java

```diff
@@ -122,17 +122,7 @@ public void appendTextToConsole(String text) {
     public void appendTextToConsole(String text, ConsoleTextType textType) {
         final Text commandText = new Text(text);
         commandText.setWrappingWidth(console.getWidth());
-        switch (textType) {
-            case NORMAL:
-                commandText.getStyleClass().addAll("consoleText", "normal");
-                break;
-            case ERROR:
-                commandText.getStyleClass().addAll("consoleText", "error");
-                break;
-            case DEFAULT:
-            default:
-                commandText.getStyleClass().addAll("consoleText", "default");
-        }
+        commandText.getStyleClass().addAll("consoleText", textType.toString());
```

Comment:
toString should never be use for anything else than logging. Can you create a separate method?